test/ngAnimate/animateSpec.js

```diff
@@ -121,6 +121,22 @@ describe("animations", function() {
       });
     });
 
+    it('should complete the leave DOM operation incase the classNameFilter fails', function() {
+      module(function($animateProvider) {
+        $animateProvider.classNameFilter(/memorable-animation/);
+      });
+      inject(function($animate, $rootScope, $document, $rootElement) {
```

Comment:
`$document` and `$rootElement` seem to be unnecessary here (in the previous test too).